demo/demo.go

```diff
@@ -0,0 +1,38 @@
+package main
+
+import (
+	"fmt"
+
+	dcrlibwallet "github.com/raedahgroup/dcrlibwallet"
+)
+
+func main() {
```

Comment:
Let's not check this file into the repo. Testing should be done with unit test files instead.